other/concat-exports.js

```diff
@@ -0,0 +1,45 @@
+const appendToFile = require('fs').appendFileSync
+const resolve = require('path').resolve
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+function capitalize(s) {
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
+}
+
+function dashToCamelCase(s) {
+  return s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+}
+
+function isCapitalized(s) {
+  return !!s && s[0] === s[0].toUpperCase()
+}
+
+// node version <= 7 does not support trailing commas in function calls
+// prettier-ignore
+const jsBuiltInConstructorNames = Object.getOwnPropertyNames(global)
+  .filter(isCapitalized)
+
+function unCollide(name) {
+  const suffix = 'Tag'
+  return jsBuiltInConstructorNames.includes(name) ? `${name}${suffix}` : name
+}
+
+const componentExports = htmlTagNames
+  .concat(svgTagNames)
+  .filter((tag, index, array) => array.indexOf(tag) === index)
+  .map(tag => {
+    const validJsName = dashToCamelCase(tag)
+    const capitalName = capitalize(validJsName)
+    // add postfix if name collides with js constructors
+    const tagName = unCollide(capitalName)
+    return `export const ${tagName} = glamorous['${tag}'];`
```

Comment:
I've considered this, and I'm not totally against it. I think that it could make things even more complex from both an API and implementation standpoint, but I'm open to talk about it in another issue if you'd like to bring it up.